chalice/utils.py

```diff
@@ -7,6 +7,27 @@
 from typing import IO, Dict, Any  # noqa
 
 
+def remove_stage_from_deployed_values(key, filename):
+    # type: (str, str) -> None
+    """Delete a top level key from the deployed JSON file."""
+    final_values = {}  # type: Dict[str, Any]
+    try:
+        with open(filename, 'r') as f:
+            final_values = json.load(f)
+    except IOError:
```

Comment:
Yea, this was me struggling to get codecov to accept that I was writing to a file. It didn't work, I'll put it back to `os.path.isfile()`.